src/main/frontend/handler/editor.cljs

```diff
@@ -1999,6 +2008,65 @@
          (db/refresh! repo {:key :block/insert :data new-blocks})
          (last metadata-replaced-blocks))))))
 
+(defn- tree->vec-tree
+  "tree:
+  [
+  {
+    :content 'this is a block',
+    :props {\"key\" \"value\" \"key2\" \"value2\"},
+    :children [
+      { :content 'this is child block' }
+    ]
+  },
+  {
+    :content 'this is sibling block'
+  }
+  ]"
+  [tree]
+  (into []
+        (mapcat
+         (fn [e]
+           (let [e* (select-keys e [:content :props])]
```

Comment:
I think we should rename `:props` to `:properties` to be consistent with the database schema.

Reply:
that makes sense
```ts
type IBatchBlock = { content: string, properties?: Record<string, any>, children?: Array<IBatchBlock> }
```

Reply:
fixed